insuree/schema.py

```diff
@@ -100,17 +100,12 @@ class Query(graphene.ObjectType):
     insuree_number_validity = graphene.Field(
         graphene.Boolean,
         insuree_number=graphene.String(required=True),
-        new_insuree=graphene.Boolean(required=False),
         description="Checks that the specified insuree number is valid"
     )
 
     def resolve_insuree_number_validity(self, info, **kwargs):
-        errors = validate_insuree_number(
-            kwargs['insuree_number'], kwargs.get('new_insuree', False))
-        if errors:
-            return False
-        else:
-            return True
+        errors = validate_insuree_number(kwargs['insuree_number'])
```

Comment:
Shouldn't information about uuid be passed in here? I don't think it'll work for insuree that's updated

Reply:
It worked because it was used when creating an insuree. The update method has uuid check. However now it is all validated in one place in InsureeService.create_or_update().

In **resolve_validate_insuree_number** uuid is not required since it is only used in frontend validation which won't validate if number belongs to updated object.